src/style.css

```
@@ -1,3 +1,68 @@
+* {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  position: relative;
+}
+
+.logo {
+  padding-top: 2px;
+}
+
+.nav__list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  margin-right: 20px;
+  position: relative;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link,
+a[data-qa="hover"] {
+  display: block;
+  text-decoration: none;
+  color: black;
+  text-align: center;
+  background-color: transparent;
+  font-size: 12px;
+  padding: 23px 0;
+}
+
+.nav__link:hover,
+a[data-qa="hover"]:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
```

and move repeated colors to variables

```suggestion
.nav__link:hover,
.is-active {
  color: $accent-color;
}
```